src/impl/metadata/retriever/profileRetriever.ts

```diff
@@ -451,7 +451,7 @@ export default class ProfileRetriever extends BaseMetadataRetriever<
     return undefined;
   }
 
-  public async writeProfile(profileObj: Profile, filePath: string) {
+  public static async writeProfile(profileObj: Profile, filePath: string) {
```

Comment:
Lets move this out to shared if it used by profile Retrieve and Profile Diff..  Otherwise code gets so convoluted